src/builder.rs

```diff
@@ -164,6 +174,21 @@ impl<'ctx> Builder<'ctx> {
 
         let c_string = to_c_str(name);
         let mut args: Vec<LLVMValueRef> = args.iter().map(|val| val.as_value_ref()).collect();
+
+        #[cfg(feature = "llvm14-0")]
```

Comment:
I know it's really ugly but we should future proof these by inverting both this and below with the full expansion:

`#[cfg(not(any(feature = "llvm4-0", ..., feature = "llvm13-0"))]`

and

`#[cfg(any(feature = "llvm4-0", ..., feature = "llvm13-0")]`

respectively. That way future llvm versions will be automatically supported by this code. As you noted, rust doesn't support macro attributes on statements yet. Been waiting a long time for that feature :(

cc: https://github.com/rust-lang/rust/issues/54727

Reply:
That's right, I fixed it.